app/models/agents/data_output_agent.rb

```diff
@@ -114,11 +140,12 @@ def feed_link
     end
 
     def feed_url(options = {})
-      feed_link + Rails.application.routes.url_helpers.
-                  web_requests_path(agent_id: id || ':id',
-                                    user_id: user_id,
-                                    secret: options[:secret],
-                                    format: options[:format])
+      interpolated['template']['self'].presence ||
```

Comment:
Maybe we should add an example in the `description` or wiki of what this might be set to when using `push_hubs`?